src/compile/common.ts

```diff
@@ -103,7 +103,9 @@ export function formatMixins(model: Model, channel: Channel, format: string) {
   let def: any = {};
 
   if (fieldDef.type === TEMPORAL) {
```

Comment:
Another question is do we ever need `formatType` when `format` is not specified.